recipes/parsl/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "parsl" %}
+{% set version = "0.7.2" %}
+{% set sha256 = "4fe2df4d2e49aaa6f03eab6a3798f25882cd08253df2246691a95a24ab9f7db0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Parsl/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux or py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pyzmq>=17.1.2
```

There should be a space here:
```suggestion
    - pyzmq >=17.1.2
```